Source/SwiftLintFramework/Helpers/SwiftDeclarationAttributeKindsGroup.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  SwiftDeclarationAttributeKindsGroup.swift
+//  swiftlint
+//
+//  Created by Daniel.Metzing on 27.03.18.
+//  Copyright © 2018 Realm. All rights reserved.
+//
+
+import Foundation
+import SourceKittenFramework
+
+public enum SwiftDeclarationAttributeKindsGroup: Int {
+    case acl
+    case setterACL
+    case mutators
+    case memoryReference
+    case `override`
+    case objcInteroperability
+    case availability
+    case final
+    case interfaceBuilder
+    case typeMethods
+
+    var swiftDeclarationAttributeKinds: [SwiftDeclarationAttributeKind] {
+        switch self {
+        case .acl:
+            return [.private,
+                    .internal,
+                    .public,
+                    .open]
+        case .setterACL:
+            return [.privateSetter,
```

Comment:
is open setter a thing?

Reply:
fileprivate?

Reply:
open set is a thing 🤯
<img width="248" alt="screen shot 2018-03-28 at 21 36 15" src="https://user-images.githubusercontent.com/6389088/38051978-26dcf52e-32d0-11e8-9f55-58794222e58a.png">

Reply:
added these cases too.